### PR TITLE
Changes to the non-uk video transcript

### PR DESCRIPTION
**Pull request by Sarah-DfE**
### Trello card
https://trello.com/c/zcF9fftf/6136-re-post-non-uk-video

### Context
Updated the non-uk video transcript

### CODE REVIEW

**Comment by Sarah-DfE** [comment]
@gemmadallmandfe I've created this PR for the video transcript changes (I've incorporated the changes we made on the original new non-uk pages PR), not sure how to embed the video though?

File: app/views/content/non-uk-teachers/why-train-to-teach-in-england-video-transcript.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Why train to teach in England video transcript
+description: Full transcript for the 'Why train to teach in England' video.
+
+---
+
+## Teach in England
+###Why train to teach in England?
```
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/why-train-to-teach-in-england-video-transcript.md:8]:
```suggestion
### Why train to teach in England?
```

**Review by Sarah-DfE — COMMENTED** [review]
@gemmadallmandfe made a mini change to his as one of the headings needed a space

File: app/views/content/non-uk-teachers/why-train-to-teach-in-england-video-transcript.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: Why train to teach in England video transcript
+description: Full transcript for the 'Why train to teach in England' video.
+
+---
+
+## Teach in England
+### Why train to teach in England?
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/why-train-to-teach-in-england-video-transcript.md:8]: @Sarah-DfE This isn't quite right
image

I think you just need to add a line in between - here's an example from the choosing a course page:
image

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**